ob_v3p0/impl/.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+    "trailingComma": "es5",
+    "semi": false,
+    "singleQuote": true,
+    "printWidth": 80,
+    "tabWidth": 4,
+    "proseWrap": "always"
+}
```

Comment:
Note this `.prettierrc.json` will be removed before the final pull request. As a separate issue, a `.editorconfig` or similar may be added to the repository, consistently with 1EdTech architect recommendations for all next-generation 1EdTech specs.

Reply:
...begin spaces vs tabs argument?